tensorflow/python/data/experimental/ops/threading_options.py

```diff
@@ -52,6 +54,14 @@ class ThreadingOptions(options.OptionsBase):
       "The value 0 can be used to indicate that the threadpool size should be "
       "determined at runtime based on the number of available CPU cores.")
 
+  def _has_default_values(self):
```

Comment:
change the name to `_has_non_default_values()` and the logic accordingly

Reply:
addressed in the commit.